### PR TITLE
text-to-image: Use Cases, Useful Resources

### PR DESCRIPTION
**Pull request by ishandutta0098**
- Added 4 use cases
- Added 4 useful resources 
  
Refs: #364

### CODE REVIEW

File: tasks/src/text-to-image/about.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Diverse design patterns can be generated to obtain unique fashion. This will make it easy for designers to conceptualize their design before actually implementing it.
+  
+### Architecture Industry
+  
+Architects can utilise the models to construct an environment based out on the requirements of the floor plan. This can also include the furniture that has to be placed in that environment.
+  
+## Useful Resources
+- [MinImagen - Build Your Own Imagen Text-to-Image Model](https://www.assemblyai.com/blog/minimagen-build-your-own-imagen-text-to-image-model/)
+- [OpenAI Blog - Dall E](https://openai.com/blog/dall-e/)
+- [Text to Image](https://towardsdatascience.com/text-to-image-a3b201b003ae)
```
Comment (merveenoyan) [inline tasks/src/text-to-image/about.md:22]: I'd rather remove personal blogs but keep blog from openai & assembly as they're subjected to technical writing checks & more 🙂

**Review by merveenoyan — COMMENTED** [review]
Overall looks very good to me, left a minor comments, thanks a lot for the contribution ❤️

File: tasks/src/text-to-image/about.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## Use Cases
+
+### Data Generation
+  
+Businesses can create additional data for their task specific to their use cases by inputting text and getting image outputs. 
+  
+### Immersive Conversational Chatbots
+  
+Chatbots can be made more immersive if they provide contextual images based on the input provided by the user.
```
[the remark this answers is not in the document]
Comment (ishandutta0098) [reply tasks/src/text-to-image/about.md:9]: I haven't experienced this yet, but I have read it in an article!

File: tasks/src/text-to-image/about.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  
+### Creative Ideas for Fashion Industry
+  
+Diverse design patterns can be generated to obtain unique fashion. This will make it easy for designers to conceptualize their design before actually implementing it.
+  
+### Architecture Industry
+  
+Architects can utilise the models to construct an environment based out on the requirements of the floor plan. This can also include the furniture that has to be placed in that environment.
```
[the remark this answers is not in the document]
Comment (ishandutta0098) [reply tasks/src/text-to-image/about.md:17]: Yes, some examples are also present in the open ai blog that I have added in the resources section.

File: tasks/src/text-to-image/about.md (hunk 1 above)
Comment (ishandutta0098) [reply tasks/src/text-to-image/about.md]: I have removed the personal blogs from the list.

**Comment by merveenoyan** [comment]
Looks good to me! I will wait a little for someone else to add more data (summary, model etc) before replacing it with the placeholder.
Also, we are going to add your hugging face handle at the end of page as a contributor, can you give me your handle? 🙂

**Comment by ishandutta0098** [comment]
Hi @merveenoyan, thanks for the merge! It was my first open source contribution and I look forward to many more.

My huggingface handle is `ishandutta`

**Pull request merged**